src/llm/http_llm_calculator.cc

```diff
@@ -84,8 +84,8 @@ class OpenAIChatCompletionsRequest {
         doc(doc),
         endpoint(endpoint) {}
 
-    GenerationConfig createGenerationConfig() const {
-        GenerationConfig config;
+    ov::genai::GenerationConfig createGenerationConfig() const {
```

Comment:
I thinkg its better to use ov::genai namespace for classes because it more readable and gives information about what it the origin of the class. Also ov::genai is short so it does not polute the file.